tools/pythonpkg/src/pyconnection.cpp

```diff
@@ -299,6 +301,23 @@ unique_ptr<DuckDBPyRelation> DuckDBPyConnection::Values(py::object params) {
 	return make_unique<DuckDBPyRelation>(connection->Values(values));
 }
 
+unique_ptr<DuckDBPyRelation> DuckDBPyConnection::ValuesRows(py::list params) {
+	if (!connection) {
+		throw ConnectionException("Connection has already been closed");
+	}
+	if (params.is_none()) {
+		params = py::list();
+	}
+	if (!py::hasattr(params, "__len__")) {
```

Comment:
This isn't enough to determine if something is `iterable` (`__getitem__` or `__iter__` is also required) - you can probably do `py::isinstance<py::iterable>(...)` instead

Reply:
It's optional only because I've copied over from Values' definition. I'm guessing the idea was to get an empty row but I wouldn't know what that was for.

I've replaced the checks with py::isinstance and I've also applied it to each of the items of the iterable (value_rows shouldn't be used in the same manner as values is, so passing in a list of values will fail)